src/tests/test_buildercore_terraform.py

```diff
@@ -128,6 +132,11 @@ def test_fastly_template_complex(self):
             data
         )
 
+    def test_write_template(self):
```

Comment:
Also tests read?

Reply:
Indeed, test names should be scenarios, not components